Automattic/wp-calypso
# Free hosting trial: Remove A/B test

Related to pdKhl6-2T3-p2#comment-4921.

## Proposed Changes

This PR removes the A/B test while making the free hosting trial available for all folks browsing `/setup/new-hosted-site` that are eligible for a hosting trial (i.e., new site and/or never had a trial before.)

We’ve already had a couple of use cases where it would’ve been beneficial to have it enabled to 100% (agency pitches, Reddit ads), and the cash sales and support impact are not drastically different, so it's not harmful.

## Testing Instructions

In a fresh account, navigate to `/setup/new-hosted-site` and verify that the "Try for free" CTA shows up in the plans step. 

Start the free trial and verify that browsing `/setup/new-hosted-site` again hides the "Try for free" CTA.
